tensorflow/core/platform/env.h

```diff
@@ -279,7 +283,7 @@ class Env {
   // "version" should be the version of the library or NULL
   // returns the name that LoadLibrary() can use
   virtual string FormatLibraryFileName(const string& name,
-      const string& version) = 0;
+                                       const string& version) = 0;
```

Comment:
Would prefer to leave these formatting changes to a different patch if possible.

Reply:
Done. I blame clang-format for being too smart :)

Reply:
FWIW I use the git-clang-format script to clang-format my changes without reformatting the whole file.  https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/git-clang-format

Reply:
Thanks for the tips!